api/app/controllers/spree/api/v2/storefront/account_controller.rb

```diff
@@ -14,6 +20,22 @@ def resource
           def resource_serializer
             Spree::Api::Dependencies.storefront_user_serializer.constantize
           end
+
+          def scope
+            model_class.all
```

Comment:
@KacperMekarski we cannot call `.all` this will load all user records to memory, why do we even need the scope method here?

Reply:
indeed, this method was unnecessary, removed it.